### PR TITLE
Fix tests

### LINKED ISSUE #120: New version of gcs fake breaks tests
**Describe the bug**
The new version of fsouza/fake-gcs-server (1.19) breaks plugin tests.

**To Reproduce**
Just clone master, start docker-compose and run tests.

**Expected behavior**
Tests passed.

**Desktop (please complete the following information):**
 - OS: MacOs catalina

### PR DESCRIPTION
**Pull request by pastushenkoy**
Please read [CONTRIBUTING.md](CONTRIBUTING.md) for details on our [code of conduct](CODE_OF_CONDUCT.md), and the process for submitting pull requests to us.

Fixes #120

## Proposed Changes

-
-
-

### Description

Specifies exact version of GCS mock in order to keep tests green.

### Checklist

- [x] Read the [**CONTRIBUTING**](CONTRIBUTING.md) document.
- [x] Read the [**CODE OF CONDUCT**](CODE_OF_CONDUCT.md) document.
- [ ] Add tests to cover changes.
- [x] Ensure your code follows the code style of this project.
- [x] Ensure CI and all other PR checks are green OR
    - [x] Code compiles correctly.
    - [ ] Created tests which fail without the change (if possible).
    - [x] All new and existing tests passed.
- [ ] Add your changes to `Unreleased` section of [CHANGELOG](CHANGELOG.md).
- [x] Improve and update the [README](README.md) (if necessary).
- [x] Ensure [documentation](./DOCS.md) is up-to-date. The same file will be updated in [plugin index](https://github.com/drone/drone-plugin-index/blob/master/content/meltwater/drone-cache/index.md) when your PR is accepted, so it will be available for end-users at http://plugins.drone.io.

### CODE REVIEW

**Comment by pastushenkoy** [comment]
@kakkoyun Can you help me with the build? I can't get what it wants.

**Comment by kakkoyun** [comment]
@pastushenkoy It's totally unrelated to your changes. I have fixed the master. Please rebase and it should be all good.

**Comment by pastushenkoy** [comment]
I'm afraid I already got the latest changes in my master branch.

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Fixes [#88](https://github.com/meltwater/drone-cache/issues/88).
 - [#86](https://github.com/meltwater/drone-cache/pull/86) Fixed rebuild errors when a nested path given in mount.
   - Fixes [#87](https://github.com/meltwater/drone-cache/issues/87).
+- [120](https://github.com/meltwater/drone-cache/pull/121)
```
Comment (kakkoyun) [inline CHANGELOG.md:29]: I don't think we need a changelog entry for this one. Just external dependency update for tests.

**Comment by kakkoyun** [comment]
I've forgotten to push. Now it's fixed and your branch is up-to-date. If you would address the comment then we can go ahead and merge it. Thanks a lot for fixing it 😄

**Comment by pastushenkoy** [comment]
Seems like it's done? )

**Pull request merged**